puppetry/__init__.py

```diff
@@ -15,16 +15,16 @@
 to the process's stdin while receiving messages the script writes to its stdout.
 
 The messages have the form:
-num_bytes:{pump="puppetry",data='{"joint_name":{"param_name":[r1.23,r4.56,r7.89]}, ...}"'}
+num_bytes:{pump='puppetry',data='{'joint_name':{'param_name':[r1.23,r4.56,r7.89]}, ...}''}
```

Comment:
I realize this is a comment, but the `data` key should hold a `dict`, not a string representation of a dict; in other words it should say `data={...}` instead of `data='{...}'` (or `data='{...}''` as it seems).